packages/grafana-ui/src/components/Table/TableCell.tsx

```diff
@@ -16,7 +16,7 @@ export const TableCell: FC<Props> = ({ cell, tableStyles, onCellFilterAdded, col
   const cellProps = cell.getCellProps();
   const field = (cell.column as any as GrafanaTableColumn).field;
 
-  if (!field.display) {
+  if (!field?.display) {
```

Comment:
This will have the table show an empty row rather than exploding if field is not defined.  I think that is a better behavior :)